### PR TITLE
fix issue 48

### PR DESCRIPTION
**Pull request by eemhu**
Adds more checks in JsonRecords to allow all records to pass through

Fixes issue
* #48

### CODE REVIEW

File: src/main/java/com/teragrep/aer_02/json/JsonRecords.java [hunk 1]
```diff
@@ -78,9 +78,14 @@ public String[] records() {
             return rv;
         }
 
-        final JsonValue recordsStructure = mainStructure.getValue("/records");
```
Comment (MoonBow-1) [inline src/main/java/com/teragrep/aer_02/json/JsonRecords.java:81]: Was there a reason why this has a slash before the "records"?

**Review by MoonBow-1 — COMMENTED** [review]
A comment about the previous implementation

File: src/main/java/com/teragrep/aer_02/json/JsonRecords.java (hunk 1 above)
Comment (eemhu) [reply src/main/java/com/teragrep/aer_02/json/JsonRecords.java:81]: getValue() uses JSONPointer, and it uses `/` as a separator. Like `/list/0` would fetch first item in list.
get() just directly accesses the underlying map, returning null if no key is present.

Comment (MoonBow-1) [reply src/main/java/com/teragrep/aer_02/json/JsonRecords.java:81]: Thanks for letting me know about this change!

**Review by MoonBow-1 — APPROVED** [review]
LGTM! Tests check for this new behavior

**Review by kortemik — APPROVED** [review]

**Pull request merged**